lms/djangoapps/grades/api/v1/gradebook_views.py

```diff
@@ -778,3 +779,159 @@ def _log_update_result(
+              and not graded problems
+            * earned_graded_override: The float overriden grade a user earned for only graded problems
+            * possible_graded_override: The float overriden highest possible grade a user can earn
+              for only graded problems
+        * history: A list of history objects that contain
+            * user: The string representation of the user who was responsible for overriding the grade
+            * comments: A string comment about why that person changed the grade
+            * created: The date timestamp the grade was changed
+            * feature: The string representation of the feature through which the grade was overrriden
+            * action: The string representation of the CRUD action the override did
+
+        An HTTP 404 may be returned for the following reasons:
+            * The requested subsection_id is invalid.
+            * The requested user_id is invalid.
+            * NOTE: if you pass in a valid subsection_id and a valid user_id with no data representation in the DB
+              then you will still recieve a 200 with a response with 'original_grade', 'override' and 'course_id'
+              set to None and the 'history' list will be empty.
+
+    **Example GET Response**
+        {
+            "subsection_id": "block-v1:edX+DemoX+Demo_Course+type@sequential+block@basic_questions",
+            "user_id": 2,
+            "course_id": "course-v1:edX+DemoX+Demo_Course",
+            "original_grade": {
+                "earned_all": 0,
+                "possible_all": 11,
+                "earned_graded": 8,
+                "possible_graded": 11
+            },
+            "override": {
+                "earned_all_override": null,
+                "possible_all_override": null,
+                "earned_graded_override": 8,
+                "possible_graded_override": null
+            },
+            "history": [
+                {
+                    "user": "edx",
+                    "comments": null,
+                    "created": "2018-12-03T18:52:36.087134Z",
+                    "feature": "GRADEBOOK",
+                    "action": "CREATEORUPDATE"
+                },
+                {
+                    "user": "edx",
+                    "comments": null,
+                    "created": "2018-12-03T20:41:02.507685Z",
+                    "feature": "GRADEBOOK",
+                    "action": "CREATEORUPDATE"
+                },
+                {
+                    "user": "edx",
+                    "comments": null,
+                    "created": "2018-12-03T20:46:08.933387Z",
+                    "feature": "GRADEBOOK",
+                    "action": "CREATEORUPDATE"
+                }
+            ]
+        }
+    """
+
+    def get(self, request, subsection_id):
+        """
+        Returns a gradebook entry/entries (i.e. both course and subsection-level grade data)
```

Comment:
This can be fixed in a subsequent commit if all of the tests pass: this is copypasta from the gradebook get() method.